docker/generic/start_proxy.py

```diff
@@ -160,6 +160,11 @@ def make_argparser():
     HTTP/2 secure connections on listener_port. Requires the certificate and
     key files "server.crt" and "server.key" within this path.''')
 
+    parser.add_argument('--ssl_backend_path', default=None, help='''
```

Comment:
should we call "ssl_client_path" since the other one is called "ssl_server_path"?

Reply:
well, then we need another flag to ssl_backend_mtls to indicate backend need mutual tls, not service config. 
OK,  it really mean ssl_client_cert path for backend.

Reply:
Changed to ssl_client_cert_path.